x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -16,6 +16,7 @@ export const APP_ICON = 'securityAnalyticsApp';
 export const APP_ICON_SOLUTION = 'logoSecurity';
 export const APP_PATH = `/app/security`;
 export const ADD_DATA_PATH = `/app/home#/tutorial_directory/security`;
+export const ADD_INTEGRATION_PATH = `/app/integrations/browse?category=security`;
```

Comment:
I think the advice was to switch to this method:
```suggestion
import {pagePathGetters} from '../path/to/fleet/plugin/public'
export const ADD_INTEGRATION_PATH = pagePathGetters.integration_all({ category: 'security'};
```